modules/eventing/pages/eventing-api.adoc

```diff
@@ -72,10 +90,62 @@ Sample API:
 
 ----
 curl -XPOST -d '{"deployment_status":false,"processing_status":false}'
-http://Administrator@192.168.1.5:8096/api/v1/functions/[sample_name]/settings
+http://Administrator:password@192.168.1.5:8096/api/v1/functions/[sample_name]/settings
+----
+
+| POST
+| [.path]_/api/v1/functions/[function_name]/settings_
+a|
+Updates an undeployed a Function with the provided setting. Do not update settings for a deployed function.
+Note you must always specify deployment_status (deplyed/undeployed) and processing_status (they must match in 6.0) when using this REST endpoint to opdate any option(s).
+
+Sample API (alter worker_count):
+
+----
+curl -XPOST -d '{"deployment_status":false,"processing_status":false,"worker_count":6}'
+http://Administrator:password@192.168.1.5:8096/api/v1/functions/[sample_name]/settings
 ----
 
 | POST
 | [.path]_/api/v1/functions/[function_name]/settings_
-| Deploys a Function with the provided code.
+a|
+Updates an undeployed a Function with the provided settings. Do not update settings for a deployed.
```

Comment:
Remove extra article: "Updates an undeployed Function with..."
Fix incomplete sentence: "Do not update settings for a deployed Function"